app/models/job.rb

```diff
@@ -10,13 +10,24 @@ class Job < ActiveRecord::Base
 
   after_update :notify_if_published
 
-  scope :published, lambda {
-    where("jobs.published_at IS NOT NULL AND jobs.published_at <= ?", Time.zone.now)
-  }
+  class << self
+    # Jobs that should appear in public listings
+    #
+    # @returns [ActiveRecord::Relation<Job>] active jobs
 
-  scope :stale, lambda {
-    where("created_at < ?", 60.days.ago)
-  }
+    def active
+      published.where("jobs.published_at >= ?", 60.days.ago)
+    end
+
+    private
+
```

Comment:
The main reason I went with defining class methods instead of scopes was that I wanted to make "published" private to indicate it wasn't used outside the Job class now; that frees future us up to change it knowing that it's not a part of Job's public API. I also wanted to add documentation to the "active" method and that looked better to me with it as a method.

These stylistic preferences are different from what was there before (just using scopes), so let me know if you would prefer scopes since that was the existing style!